aws/email.go

```diff
@@ -0,0 +1,75 @@
+package aws
+
+import (
+	"bytes"
+	encsv "encoding/csv"
+	amaws "github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/ses"
+	c "github.com/companieshouse/innovation-accounts-statistics-tool/config"
+	"github.com/companieshouse/innovation-accounts-statistics-tool/models"
+	"gopkg.in/gomail.v2"
+	"io"
+)
+
+const (
+	subject   = "Small Full Accounts Statistics"
+	body      = "<h1>SFA Stats</h1><p>Attached is the CSV of the stats</p>"
+	awsRegion = "eu-west-1"
```

Comment:
Shall we make this configurable so it can be run in any region (once provisioned)